### PR TITLE
README.ES.md --> 03

### PR DESCRIPTION
**Pull request by zanezhub**
# 03 Hacky Hello, World
I just translated the README.md of the third tutorial.

### CODE REVIEW

File: 03_hacky_hello_world/README.ES.md [hunk 1]
```diff
@@ -0,0 +1,319 @@
+# Tutorial 03 - Hacky Hello World
+
+## tl;dr
+
+* Introduciendo la macro global `print!()` para habilitar "printf debugging" lo más pronto posible.
```
Comment (eldruin) [inline 03_hacky_hello_world/README.ES.md:5]:
```suggestion
* Se añade la macro global `print!()` para habilitar la "depuración basada en printf" lo más pronto posible.
```

File: 03_hacky_hello_world/README.ES.md [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+* Para mantener una duración razonable en este tutorial, imprimir funciones por el momento "abusa" una propiedad de QEMU que nos permite hacer uso del `UART` de la Raspberry sin haberla configurado apropiadamente.
```
Comment (eldruin) [inline 03_hacky_hello_world/README.ES.md:6]:
```suggestion
* Para mantener una duración razonable en este tutorial, las funciones de impresión "abusan" por el momento una propiedad de QEMU que nos permite hacer uso del `UART` de la Raspberry sin haberla configurado apropiadamente.
```

File: 03_hacky_hello_world/README.ES.md [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+* Usar el hardware real de `UART` se habilitará paso por paso en los siguientes tutoriales.
```
Comment (eldruin) [inline 03_hacky_hello_world/README.ES.md:7]:
```suggestion
* El uso del hardware real de `UART` se habilitará paso por paso en los siguientes tutoriales.
```

File: 03_hacky_hello_world/README.ES.md [hunk 4]
```diff
@@ -0,0 +1,319 @@
...
+
+## Adiciones notables
+
+* `src/console.rs` introduce la interfaz `Traits` para comandos de consola.
```
Comment (eldruin) [inline 03_hacky_hello_world/README.ES.md:11]:
```suggestion
* `src/console.rs` introduce una interfaz con `trait`s para comandos de consola.
```

File: 03_hacky_hello_world/README.ES.md [hunk 5]
```diff
@@ -0,0 +1,319 @@
...
+* `src/bsp/raspberrypi/console.rs` implementa la interfaz para que QEMU pueda crear una emulación de UART.
+* El *panic handler* (manipulador de pánico) hace uso de la nueva macro `print!()` para mostrar mensajes de error del usuario.
```
Comment (eldruin) [inline 03_hacky_hello_world/README.ES.md:13]:
```suggestion
* El *panic handler* (manejador de pánico) hace uso de la nueva macro `print!()` para mostrar mensajes de error del usuario.
```

File: 03_hacky_hello_world/README.ES.md [hunk 6]
```diff
@@ -0,0 +1,319 @@
...
+* Hay un nuevo objetivo Makefile, `make test`, destinado a la automatización de pruebas. Este comando inicia el kernel (núcleo) compilado en `QEMU`, y busca una cadena específica que ha sido producida por el kernel (núcleo).
```
Comment (eldruin) [inline 03_hacky_hello_world/README.ES.md:14]:
```suggestion
* Hay un nuevo objetivo en el Makefile: `make test`, destinado a la automatización de pruebas. Este comando inicia el kernel (núcleo) compilado en `QEMU`, y busca una cadena de texto específica en la salida que ha sido producida por el kernel (núcleo).
```

File: 03_hacky_hello_world/README.ES.md [hunk 7]
```diff
@@ -0,0 +1,319 @@
...
+Hello from Rust!
+
+Kernel panic: Stopping here.
+```
+
+### Diccionario
+
+* *Hacky:* Solución torpe o poco elegante para un problema.
+* *Debugging:* Proceso para identificar y corregir errores de programación.
+  * *printf debugging:* Usado para describir el trabajo de *debugging* poniendo comandos que dan una salida en consola, como el de "printf", en diferentes lugares del programa; observando la información y tratando de deducir qué está mal en el programa basandose en la información que nos da nuestros comandos.
```
Comment (eldruin) [inline 03_hacky_hello_world/README.ES.md:33]:
```suggestion
  * *printf debugging:* Usado para describir el trabajo de depuración (*debugging*) poniendo comandos que dan una salida en consola, como el de "printf", en diferentes lugares del programa; observando la información y tratando de deducir qué está mal en el programa basándose en la información que nos dan nuestros comandos.
```

**Review by eldruin — COMMENTED** [review]
Thanks! Just a few corrections/improvements.

**Comment by zanezhub** [comment]
I added the corrections/improvements in 6b17b91.
Thanks

**Review by eldruin — APPROVED** [review]
Looks good to me, thank you!

**Pull request merged**